Program.cs

```diff
@@ -253,6 +253,58 @@ static void CityMenu(Statistics cityStats)
                         Console.WriteLine("Press any key to continue");
                         Console.ReadKey();
                         break;
+                    case "7":
+                        bool valid = false;
+                        string fileName = "";
+                        ulong newPopulation = 0;
+                        Console.WriteLine("Available data files:\n");
+                        Console.WriteLine("\t1) Canadacities-JSON.json");
+                        Console.WriteLine("\t2) Canadacities-XML.xml");
+                        Console.WriteLine("\t3) Canadacities.csv");
+                        Console.Write("Please make a selection(ex. 1, 2): ");
+                        do
+                        {
+                            selection = Console.ReadLine();
+                            switch (selection)
+                            {
+                                case "1":
+                                    fileName = "Canadacities-JSON.json";
+                                    valid = true;
+                                    break;
+                                case "2":
+                                    fileName = "Canadacities-XML.xml";
+                                    valid = true;
+                                    break;
+                                case "3":
+                                    fileName = "Canadacities.csv";
+                                    valid = true;
+                                    break;
+                                default:
+                                    Console.Write("\nInvalid selection, please enter a valid selection: ");
+                                    break;
+                            }
+                        } while (!valid);
+                        Console.WriteLine("\nWhich city do you want to change the population?\n");
+                        //get the user's choice
+                        response = OneCityValidator(cityStats);
+                        do
+                        {
+                            Console.Write($"Please enter the new population for {response}: ");
+                            if (!UInt64.TryParse(Console.ReadLine(), out newPopulation))
+                            continue;
+                            else break;
+                        }
+                        while (true);
+                        try
+                        {
+                            cityStats.UpdatePopulation(response, newPopulation, fileName);
+                        }catch (Exception ex)
+                        {
+                            Console.WriteLine($"ERROR: {ex.Message}");
+                        }
+                        Console.WriteLine("Press any key to continue");
+                        Console.ReadKey();
+                        break;
```

Comment:
Maybe you have a bit of an argument to put it into a new method, its a bit lengthy. If it isnt done in this pr I say we do it in the next